src/components/app.js

```diff
@@ -304,11 +318,19 @@ const App = () => {
                         appsAndUrls={appsAndUrls}
                     >
                         <Tabs
-                            value={selectedTabId}
+                            value={
+                                configs
+                                    .map((c) => c.process)
```

Comment:
we want this code only once. please find a way to get rid of the duplication with the useEffect line 250 doing setSelectedTabId(...)